dbms/src/IO/WriteBufferValidUTF8.cpp

```diff
@@ -12,26 +12,23 @@ namespace DB
 
 const size_t WriteBufferValidUTF8::DEFAULT_SIZE = 4096;
 
-namespace
+/** Index into the table below with the first byte of a UTF-8 sequence to
+  * get the number of trailing bytes that are supposed to follow it.
+  * Note that *legal* UTF-8 values can't have 4 or 5-bytes. The table is
+  * left as-is for anyone who may want to do such conversion, which was
+  * allowed in earlier algorithms.
+  */
+extern const UInt8 length_of_utf8_sequence[256] =
```

Comment:
Isn't it better to expose a function instead of variable?

PS. We have another function (`UTF8::seqLength`), that use bit-scan instruction
and I'm not sure what is better.

Reply:
I will investigate that